### PR TITLE
docs: adding bit about custom amis

### PR DESCRIPTION
**Pull request by msdolbey**
**What problem does this PR solve?**:
Just adding some more flavor/instructions for how to locate and put in a customized source_ami

**Which issue(s) does this PR fix?**:
* https://jira.d2iq.com/browse/D2IQ-94270

**Special notes for your reviewer**:
solely documentation

**Does this PR introduce a user-facing change?**:
solely documentation

### CODE REVIEW

**Comment by msdolbey** [comment]
https://teamcity.mesosphere.io/viewLog.html?buildId=3695161&buildTypeId=ClosedSource_KonvoyImageBuilder_E2eCentOS7 - this failing is why we're adding this, lol

File: docs/dev/changing-amis.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+At times, a particular upstream AMI may not be available in your region, or something could be renamed, or perhaps you want to provide a custom AMI for whatever reason you need.
+
+If this is the case, you will want to edit, or create your own, yaml file that looks up based on the `source_ami` field.
+
+For example, [CentOS also provides an image](https://wiki.centos.org/Cloud/AWS) on the [AWS marketplace](https://aws.amazon.com/marketplace/pp/prodview-foff247vr2zfw) which you can subscribe to for free.
+
+Once you select the source AMI that you want, you can declare that when running your build command:
+
+```bash
+./konvoy-image build path/to/ami/centos-79.yaml --source-ami ami-0123456789
```
Comment (supershal) [inline docs/dev/changing-amis.md:35]: We want to start promoting the provider specific subcommands. `konvoy-image build aws|azure|gcp` 
```suggestion
konvoy-image build aws  --source-ami ami-0123456789 path/to/ami/centos-79.yaml
```

**Review by supershal — COMMENTED** [review]
Thanks for writing this in doc

**Review by supershal — APPROVED** [review]
🎉

File: docs/dev/changing-amis.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+When using KIB for building machine images to Amazon, the default source AMIs that we provide are based on looking up an AMI based on the owner, and a filter for that operating system and version.
+
+You can view an example of that with the provided `centos-79.yaml` snippet below:
+
+```yaml
+download_images: true
+
+packer:
+  ami_filter_name: "CentOS 7.9.2009 x86_64"
```
Comment (mmcelroy91) [inline docs/dev/changing-amis.md:16]:
```suggestion
  ami_filter_name: "CentOS 7.9.2009 x86_64"
 
```
I think it changed to CentOS Linux 7

**Comment by supershal** [comment]
The e2e tests were broken because of broken centos AMI. rebased the branch and running the tests again before merging.

**Pull request merged**